api/src/main/java/javax/enterprise/inject/spi/configurator/ObserverMethodConfigurator.java

```diff
@@ -17,22 +17,22 @@
 
 package javax.enterprise.inject.spi.configurator;
 
+import java.lang.annotation.Annotation;
+import java.lang.reflect.Method;
+import java.lang.reflect.Type;
+import java.util.Set;
+
+import javax.enterprise.event.ObserverException;
 import javax.enterprise.event.Reception;
 import javax.enterprise.event.TransactionPhase;
 import javax.enterprise.inject.spi.AfterBeanDiscovery;
 import javax.enterprise.inject.spi.AnnotatedMethod;
-import javax.enterprise.inject.spi.EventMetadata;
+import javax.enterprise.inject.spi.EventContext;
 import javax.enterprise.inject.spi.ObserverMethod;
-import java.lang.annotation.Annotation;
-import java.lang.reflect.Method;
-import java.lang.reflect.Type;
-import java.util.Set;
-import java.util.function.BiConsumer;
-import java.util.function.Consumer;
 
 /**
- * This API is an helper to build a new {@link ObserverMethod} instance.
- * CDI container must provides an implementation of this interface accessible.
+ * This API is an helper to build a new {@link ObserverMethod} instance. CDI container must provides an implementation of this
+ * interface accessible.
```

Comment:
Yep, at least `must provides` is gramatically wrong. 

I would keep it simple: `The container must provide an implementation of this interface.`?